NOTICE

```diff
@@ -35,7 +35,7 @@ Copyright (C) 2013 GoDaddy Operating Company, LLC
 
 ~~~
 
-lib/cppapi developed by LinkedIn
+include/tscpp/api developed by LinkedIn
```

Comment:
These changes are unrelated cleanup, from moving lib/cppapi to include/tscpp/api and src/tscpp/api .